arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/gas/projector/DataCopy.java

```diff
@@ -18,23 +18,19 @@
 import static org.hyperledger.besu.evm.internal.Words.clampedToLong;
 
 import net.consensys.linea.zktracer.opcode.gas.GasConstants;
-import org.apache.tuweni.bytes.Bytes;
 import org.hyperledger.besu.evm.frame.MessageFrame;
 import org.hyperledger.besu.evm.internal.Words;
 
 public final class DataCopy implements GasProjection {
   private final MessageFrame frame;
   private long offset = 0;
-  private long size = 0;
-  private int bitSize = 0;
+  private long len = 0;
 
   public DataCopy(MessageFrame frame) {
```

Comment:
Where is this method used ? Asking because the offset/size arguments won't be at the same position for `CODECOPY` as they will be for `EXTCODECOPY`